From 548b3b099d230b1221edf8062341900679d83f3e Mon Sep 17 00:00:00 2001
From: Wolfgang Bumiller <w.bumiller@proxmox.com>
Date: Thu, 10 Dec 2015 14:46:59 +0100
Subject: [PATCH 34/55] blockdev: bdrv_open dropped the drv parameter

As per commit:
6ebf9aa2ef: block: Drop drv parameter from bdrv_open()

For-patch: backup: vma: add dir format
---
 blockdev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/blockdev.c b/blockdev.c
index dd4e4bd..bf61a74 100644
--- a/blockdev.c
+++ b/blockdev.c
@@ -3374,7 +3374,7 @@ UuidInfo *qmp_backup(const char *backup_file, bool has_format,
             }
 
             di->target = bdrv_new();
-            if (bdrv_open(&di->target, di->targetfile, NULL, NULL, flags, NULL, &local_err) < 0) {
+            if (bdrv_open(&di->target, di->targetfile, NULL, NULL, flags, &local_err) < 0) {
                 bdrv_unref(di->target);
                 error_propagate(errp, local_err);
                 goto err;
-- 
2.1.4

